assets/js/script.js

```diff
@@ -0,0 +1,84 @@
+// Assignment Code
+var generateBtn = document.querySelector("#generate");
+
+// Write password to the #password input
+function writePassword() {
+  // new secure password
+  const password = generatePassword();
+  const passwordText = document.querySelector("#password");
+
+  passwordText.value = password;
+}
+
+function generatePassword() {
+  // prompt for the length of the password
+  const passLength = prompt(
+    "Please enter how many characters you would like your password to be. \nMust be between 8 - 128 characters."
+  );
+
+  const lengthConfirm = parseInt(passLength);
+  console.log(lengthConfirm);
+
```

Comment:
Please remove all the console logs from the code.

Reply:
removed console logs